helm-charts/0.26.1/charts/spire/templates/spire-server-config-map.yaml

```diff
@@ -16,71 +16,98 @@ metadata:
-      trust_domain = {{ .Values.global.spire.trustDomain | quote }}
-      data_dir = "/run/spire/server/data"
-      log_level = "DEBUG"
-      federation {
-        bundle_endpoint {
-          address = "0.0.0.0"
-          port = 8443
-        }
-      }
-
-{{- if .Values.experimental.eventsBasedCache }}
-      experimental {
-        events_based_cache = true
-      }
-{{- end }}
-
-    }
-
-    health_checks {
-      bind_address = "0.0.0.0"
-      bind_port = "8080"
-      listener_enabled = true
-      live_path = "/live"
-      ready_path = "/ready"
-    }
-
-    plugins {
-      DataStore "sql" {
-        plugin_data {
-          database_type = "sqlite3"
-          connection_string = "/run/spire/server/data/datastore.sqlite3"
-        }
-      }
-
-      NodeAttestor "k8s_psat" {
-        plugin_data {
-          clusters = {
-            "vsecm-cluster" = {
-              service_account_allow_list = ["{{ .Values.global.spire.namespace }}:spire-agent"]
+    {
+      "health_checks": {
+        "bind_address": "0.0.0.0",
+        "bind_port": "8080",
+        "listener_enabled": true,
+        "live_path": "/live",
+        "ready_path": "/ready"
+      },
+      "plugins": {
+        "DataStore": [
+          {
+            "sql": {
+              "plugin_data": {
+                "connection_string": "/run/spire/data/datastore.sqlite3",
+                "database_type": "sqlite3"
+              }
             }
           }
+        ],
+        "KeyManager": [
+          {
+            "disk": {
+              "plugin_data": {
+                "keys_path": "/run/spire/data/keys.json"
+              }
+            }
+          }
+        ],
+        "NodeAttestor": [
+          {
+            "k8s_psat": {
+              "plugin_data": {
+                "clusters": [
+                  {
+                    "vsecm-cluster": {
+                      "allowed_node_label_keys": [],
+                      "allowed_pod_label_keys": [],
+                      "audience": [
+                        "spire-server"
+                      ],
+                      "service_account_allow_list": [
+                        "spire-system:spire-agent"
+                      ]
+                    }
+                  }
+                ]
+              }
+            }
+          }
+        ],
+        "Notifier": [
+          {
+            "k8sbundle": {
+              "plugin_data": {
+                "config_map": "spire-bundle",
+                "namespace": "spire-system"
+              }
+            }
+          }
+        ]
+      },
+      "server": {
+        "audit_log_enabled": false,
+        "bind_address": "0.0.0.0",
+        "bind_port": "8081",
+        "ca_key_type": "rsa-2048",
+        "ca_subject": [
+          {
+            "common_name": "vsecm.com",
+            "country": [
+              "US"
+            ],
+            "organization": [
+              "vsecm.com"
+            ]
+          }
+        ],
```

Comment:
The `"server":` stanza has a bunch of additions, including this explicitly defined CA details.

Again, better to be explicit.